### PR TITLE
Closes AgileVentures/MetPlus_tracker#137

### PR DESCRIPTION
**Pull request by patmbolger**
added agency branch model

### CODE REVIEW

File: db/migrate/20151127164027_create_branches.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreateBranches < ActiveRecord::Migration
+  def change
+    create_table :branches do |t|
+      t.references :agency, index: true, foreign_key: true
+      t.string :code, limit: 8
+
```
Comment (joaopapereira) [inline db/migrate/20151127164027_create_branches.rb]: arent ẃe missing the address_id?

Comment (patmbolger) [inline db/migrate/20151127164027_create_branches.rb]: The Branch model has_one address, so the foreign key is in the addresses table (using the polymorphic ID and type).

Comment (joaopapereira) [inline db/migrate/20151127164027_create_branches.rb]: Ok

**Comment by joaopapereira** [comment]
Look good

**Pull request merged**